### PR TITLE
DCS-1874 bumping node and lib versions

### PR DESCRIPTION
**Pull request by andrewrlee**
Also speeding up tests and adding missing licence and adding compliance badge and moving dotenv to dev dependencies

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (andrewrlee) [inline LICENSE:1]: Adding missing licence for compliance

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Use of Force
 [![CircleCI](https://circleci.com/gh/ministryofjustice/use-of-force/tree/main.svg?style=svg)](https://circleci.com/gh/ministryofjustice/use-of-force)
-[![Known Vulnerabilities](https://snyk.io/test/github/ministryofjustice/use-of-force/badge.svg)](https://snyk.io/test/github/ministryofjustice/use-of-force)
+[![repo standards badge](https://img.shields.io/badge/dynamic/json?color=blue&style=for-the-badge&logo=github&label=MoJ%20Compliant&query=%24.result&url=https%3A%2F%2Foperations-engineering-reports.cloud-platform.service.justice.gov.uk%2Fapi%2Fv1%2Fcompliant_public_repositories%2Fuse-of-force)](https://operations-engineering-reports.cloud-platform.service.justice.gov.uk/public-github-repositories.html#use-of-force "Link to report")
```
Comment (andrewrlee) [inline README.md:3]: Adding compliance badge

File: package.json [hunk 3]
```diff
@@ -12,14 +12,15 @@
     "watch-node": "DEBUG=gov-starter-server* DB_PORT=5433 nodemon --watch dist/ dist/server.js | bunyan -o short",
     "start:dev": "npm run build && concurrently -k -p \"[{name}]\" -n \"Views,TypeScript,Node\" -c \"yellow.bold,cyan.bold,green.bold\" \"npm run watch-views\" \"npm run watch-ts\" \"npm run watch-node\"",
     "start-feature": "export $(cat feature.env) && node $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
-    "watch-node-feature": "export $(cat feature.env) && nodemon --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
+    "watch-node-feature": "export $(cat feature.env) && nodemon -r dotenv/config --watch dist/ $NODE_DEBUG_OPTION dist/server.js | bunyan -o short",
```
Comment (andrewrlee) [inline package.json:15]: dotenv is now only preloaded in dev mode rather than an application dependency

File: package.json [hunk 4]
```diff
@@ -32,7 +33,14 @@
...
   "jest": {
-    "preset": "ts-jest",
+    "transform": {
+      "^.+\\.(ts|tsx|js|jsx)$": [
+        "ts-jest",
+        {
+          "isolatedModules": true
+        }
+      ]
+    },
```
Comment (andrewrlee) [inline package.json:43]: using isolated modules to speed up tests

File: package.json [hunk 5]
```diff
@@ -103,72 +111,72 @@
...
   "devDependencies": {
     "@types/bunyan": "^1.8.8",
-    "@types/express": "^4.17.13",
+    "@types/express": "^4.17.14",
     "@types/hapi__joi": "^17.1.8",
     "@types/http-errors": "^1.8.2",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^17.0.31",
+    "@types/jest": "^29.1.1",
+    "@types/node": "^16.11.62",
```
Comment (andrewrlee) [inline package.json:148]: downgrading accidentally uploaded node versions

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**